doc/oil-overview.md

```diff
@@ -149,7 +149,7 @@ Mentioned in the last post.  This is big!
 - Like shell and Python, there is a global scope and function scope..
 - shell functions don't live in their own scope?  This is like Lisp-1 vs
   Lisp-2.
-- There won't be dynamic scope as in shell.
+- There won't be dynamic scope as in the shell.
```

Comment:
This is a fine distinction, but "in shell" to me means "in the shell language".  Sort of like "this works in Python", "this works in shell".

"the shell" sounds like something you type into, not a language.